services/EventService.ts

```diff
@@ -15,11 +15,19 @@ export default class EventService {
     this.transactions = new TransactionsManager(entityManager);
   }
 
-  public async create(event: Event) {
+  /**
+   * Creates a new event
+   *
+   * @param event object with all the properties of the event
+   * @returns The event that was created
+   */
+  public async create(event: Event): Promise<PublicEvent> {
     const eventCreated = await this.transactions.readWrite(async (txn) => {
       const eventRepository = Repositories.event(txn);
       const isUnusedAttendanceCode = eventRepository.isUnusedAttendanceCode(event.attendanceCode);
       if (!isUnusedAttendanceCode) throw new UserError('Attendance code has already been used');
+      const endAfterStartDate = event.start < event.end;
```

Comment:
nit: let's actually do isEndAfterStart to be clear that it is a boolean

Reply:
I'd argue since the if statement condition itself is fairly simple, I don't think it's necessary to create a separate variable here - we can simply inline this (additionally with a slightly clearer error message):
```ts
      if (event.start >= event.end) throw new UserError('Event has start time at or later than end time');
```

Reply:
But if it made more sense to create a separate variable, then agreed with Marcelo's comment on naming. Also, I think it'd just be clearer to do `isStartAfterEnd` and use that bool instead of doing `isEndAfterStart` and using the inverse of that bool - the inverse to me seems unnecessary.

Reply:
tbh I figured the idea was to keep the consistency of having
1. a variable that describes the condition of intended behavior
2. if check over the variable 
3. throw error if variable is false
Not sure if it makes it more clear, definitely is not the most effective code. But if we want to change endAfterStartDate we should also change isUnusedAttendanceCode since it is written in a similar format.

Reply:
Generally agreed, though as mentioned above, I think it's a bit redundant to always have to create a variable for a conditional if its already clear what the condition is supposed to do. 

> But if we want to change endAfterStartDate we should also change isUnusedAttendanceCode since it is written in a similar format.

For this specific case, isUnusedAttendanceCode is an async call so we can't really have that inlined with the if statement (right now it's wrongly not `await`ed, but #356 resolves this)